presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestSpatialQueries.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.geospatial;
+
+import org.testng.annotations.Test;
+
+public class TestSpatialQueries
+        extends AbstractTestSpatialQueries
+{
+    @Test
+    public void testUnionAll()
+    {
+        assertQuery("SELECT ST_AsText(point) FROM (" +
+                "SELECT * FROM (VALUES (ST_Point(1, 2)), (ST_Point(3, 4))) as t(point) " +
+                "   UNION ALL " +
```

Comment:
Doesn't `UNION ALL` work without the `equalTo` implemented? Or did you mean `UNION`?

Reply:
`UNION ALL` works not, but `UNION` needs hash_code operator.